src/Tizen.NUI/src/public/BaseComponents/TextConstants.cs

```diff
@@ -47,6 +47,36 @@ public struct InputFilter
         public string Rejected { get; set; }
     }
 
+
+    /// <summary>
+    /// A struct to pass data of Strikethrough PropertyMap. <br />
+    /// </summary>
+    /// <remarks>
+    /// The Strikethrough struct is used as an argument to SetStrikethrough and GetStrikethrough methods. <br />
+    /// See <see cref="Tizen.NUI.BaseComponents.TextLabel.SetStrikethrough"/>, <see cref="Tizen.NUI.BaseComponents.TextLabel.GetStrikethrough"/>, <see cref="Tizen.NUI.BaseComponents.TextField.SetStrikethrough"/>, <see cref="Tizen.NUI.BaseComponents.TextField.GetStrikethrough"/>, <see cref="Tizen.NUI.BaseComponents.TextEditor.SetStrikethrough"/> and <see cref="Tizen.NUI.BaseComponents.TextEditor.GetStrikethrough"/>. <br />
+    /// </remarks>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public struct Strikethrough
```

Comment:
Actually, I want to remove these warnings from the new structure to be added.
If we can't fix them, how about adding some messages to suppress the warnings?
e.g.
[System.Diagnostics.CodeAnalysis.SuppressMessage("Naming", "CA1815: Override equals and operator equals on value types")]

Reply:
Hi @Seoyeon2Kim 
Could you please give me more details. 
About which warnings we are talking ? I did not find them on this code.

Reply:
Helo @Seoyeon2Kim 
Thanks for your advice.

We are thinking about warnings that occur in newly added structs, but we haven't decided yet.
Perhaps it would be better to implement Equals through IEquatable and override the operator.

I think this can work on another PR after making a decision.
what do you think?

Reply:
Sorry, I found them.
You meant  warnings during build in GitHub Actions, right ?

Reply:
Yes, right. That's it.
I'll follow @wonrst 's opinion :) Let me know how to deal the warnings.

Reply:
Yes, I agree you.
We can solve it as @wonrst said in another PR
https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1815

I added SuppressMessage for Strikethrough.

@wonrst , please let us know if you decide to apply the suggested solution. We can do it :)